docs/detections/alerts-view-details.asciidoc

```diff
@@ -125,31 +155,79 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
 
 If these fields aren't available, {elastic-sec} does not perform the query and does not display threat intelligence data for the alert.
 
-Available threat indicator match data appears in the **Threat Match Detected** subsection.
+Additional threat intelligence data is generated for indicator match alerts when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. Indicator match alerts always populate the *Threat Matched Detected* section under the *Threat Intel* tab with additional threat match data.
```

Comment:
*Threat Intel* tab?

Reply:
I'll fix this, good catch! This portion of the docs was copied from "old" docs, which contain references to components in the "old" flyout.

```suggestion
Additional threat intelligence data is generated for indicator match alerts when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. Indicator match alerts always populate the *Threat Matched Detected* section under the *Threat intelligence* tab with additional threat match data.
```